core/db.py

```
@@ -7,12 +7,31 @@
 from __future__ import annotations
 
 import os
-from contextlib import contextmanager
-from typing import Any, Generator
+from contextlib import asynccontextmanager, contextmanager, suppress
```

Flake8 (7.3.0)

[error] 'contextlib.suppress' imported but unused

(F401)